tests/system/alerts/test_alerts.py

```diff
@@ -419,14 +268,17 @@ def test_job_failure_alert_sliding_window(self):
 
         # this is the first failure
         with pytest.raises(Exception):
-            self.project.run_function("test-func")
+            self.project.run_function(function_name)
 
         # Wait for more than two minutes to simulate a delay that is slightly longer than the alert period
         time.sleep(125)
 
         # this is the second failure
         with pytest.raises(Exception):
-            self.project.run_function("test-func")
+            self.project.run_function(function_name)
+
+        # wait since there is a might be a delay
+        time.sleep(20)
```

Comment:
What delay might be here?

Reply:
@TomerShor https://github.com/mlrun/mlrun/blob/5b496237e7443216e2aac17c279e2e5882e2eeb0/server/py/services/alerts/main.py#L500 which is 30s, so worst case is 30s

Reply:
so why not `time.sleep(mlconf.alerts.events_generation_interval)`

Reply:
@TomerShor not really sure if it really worth to sleep the max time every time. But alright, will do